base_deploy_kosko_stage.yml

```
@@ -46,7 +46,12 @@ variables:
           echo '---' \; -exec cat {} \; \
         >> ${CI_PROJECT_DIR}/manifest.yaml
       fi
-    - kubectl apply -f ${CI_PROJECT_DIR}/manifest.yaml
+    - |
+      if [[ -z $PRODUCTION ]]; then
+        kubectl create ns ${KUBE_NAMESPACE} || true
+        kubectl annotate ns ${KUBE_NAMESPACE} azure-pg-admin-user=sample-next-app || true
```

@gary isn't this `sample-next-app` specific ?

```suggestion
        kubectl annotate ns ${KUBE_NAMESPACE} azure-pg-admin-user=sample-next-app || true
```

---

this should be `$CI_PROJECT_NAME` ?

---

Done.